### PR TITLE
Download and install CRTM fix files.

### PR DESCRIPTION
**Pull request by aerorahul**
This PR:
- downloads and installs CRTM fix files alongside the library
- sets an environment variable `CRTM_FIX` when the `crtm` module is loaded.

### CODE REVIEW

File: libs/build_nceplibs.sh [hunk 1]
```diff
@@ -223,6 +246,17 @@ VERBOSE=$MAKE_VERBOSE make -j${NTHREADS:-4}
 [[ $USE_SUDO =~ [yYtT] ]] && sudo -- bash -c "export PATH=$PATH; make install" \
                           || make install
 
+cd ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+
+# Install CRTM fix files
+if [[ "$name" == "crtm" ]]; then
+  if [[ ${STACK_crtm_install_fix:-} =~ [yYtT] ]]; then
+    if [[ -d crtm_fix-$version ]]; then
+      ./link_crtm_coeffs.sh ./crtm_fix-$version $prefix/fix
```
Comment (kgerheiser) [inline libs/build_nceplibs.sh:255]: Does there need to be a check here if it's already installed?

Comment (aerorahul) [reply libs/build_nceplibs.sh:255]: how will it be already installed? `$prefix` is the location of the library, which is protected by `OVERWRITE`.
This is the problem remember, the coefficients living with the library.

Comment (kgerheiser) [reply libs/build_nceplibs.sh:255]: Right, the top-level overwrite is there.

**Review by kgerheiser — APPROVED** [review]

**Comment by edwardhartnett** [comment]
Thanks this is great!

**Pull request merged**